src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        StringBuilder data = new StringBuilder();
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(fileName));
+            for (String s : lines) {
+                data.append(s);
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException();
+        }
```

Comment:
catching one checked Exception just to throw another is not reasonable
you are throwing checked Exception from this method anyway and indicate this in method signature. try just removing this catch block

Reply:
understood, done